lib/datasource/common.ts

```diff
@@ -49,6 +49,10 @@ export interface ReleaseResult {
   versions?: any;
 }
 
+export interface RawRefs {
```

Comment:
Please move this to the `git-refs` file too.

Reply:
okay